WordPress/Docs/Arrays/ArrayIndentationStandard.xml

```diff
@@ -0,0 +1,52 @@
+<documentation title="Array Indentation">
+    <standard>
+    <![CDATA[
+      The array closing bracket and indentation should line up with the start of the content on the line containing the array opener.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Closing bracket lined up correctly">
+        <![CDATA[
+$args = array(
+    'post_id' => 22,
+<em>);</em>
+        ]]>
+        </code>
+        <code title="Invalid: Closing bracket lined up incorrectly">
+        <![CDATA[
+$args = array(
+            'post_id' => 22,
+        <em>);</em>
+        ]]>
+        </code>
+    </code_comparison>
+     <standard>
```

Comment:
Highly pedantic, but there are five spaces before this opening tag, so it doesn't align with the previous closing tag. Please reduce it to 4 spaces.